pkg/controller/keystone/keystone_controller.go

```diff
@@ -496,7 +496,7 @@ func newKeystoneService(cr *contrail.Keystone) *core.Service {
 	}
 }
 
-func newBootStrapJob(cr *contrail.Keystone, name types.NamespacedName, kcbName, fernetKeysSecretName, credentialKeysSecretName string, psqlIP string) *batch.Job {
+func newBootStrapJob(cr *contrail.Keystone, name types.NamespacedName, kcbName, fernetKeysSecretName, credentialKeysSecretName string, psqlIP string, nodeSelector map[string]string) *batch.Job {
```

Comment:
please remove `nodeSelector` argument since you can take it from `cr.CommonConfiguration.NodeSelector`